### PR TITLE
A cleaner merge of my changes

### PR DESCRIPTION
**Pull request by DrParadox7**
Use this instead of #35

### CODE REVIEW

**Comment by maggi373** [comment]
it fails dude

**Comment by maggi373** [comment]
a shit maven is down

**Comment by maggi373** [comment]
you need to fix the error when compiling

**Comment by DrParadox7** [comment]
Alright. Fixed the sneaking lack of ";" and now it builds.

**Comment by DrParadox7** [comment]
These commits do not include anything that can now be done with Minetweaker _(such as the steel manufacturing from Metallurgic Infuser)_ since you fixed that.

File: src/main/java/mekanism/common/integration/OreDictManager.java [hunk 1]
```diff
@@ -345,6 +344,12 @@ public static void init() {
 			InfuseRegistry.registerInfuseObject(ore, new InfuseObject(InfuseRegistry.get("DIAMOND"), 10));
 			RecipeHandler.addEnrichmentChamberRecipe(StackUtils.size(ore, 1), new ItemStack(Items.diamond));
 		}
+		
+		for(ItemStack ore : OreDictionary.getOres("gemDiamond"))
```
Comment (maggi373) [inline src/main/java/mekanism/common/integration/OreDictManager.java]: gemDiamond compat for enrichment chamber needs to be removed as its already implemented from terralizationcompat further down.

**Review by maggi373 — CHANGES_REQUESTED** [review]
gemdiamond compat needs to be removed as its already implemented from terralizationcompat

**Review by maggi373 — APPROVED** [review]

**Pull request merged**